apps/dialtone-documentation/docs/components/empty-state.md

```diff
@@ -0,0 +1,380 @@
+- <b>Body</b>:  Avoid repeating the content of the title. Also, same as the title, keep it short and to the point.
+If you need to provide a link to the user, you can use it here (e.g learn more) but please consider using the CTA section to be more concise to the user and more consistent to Dialtone.
+- <b>CTA</b>:  We recommend the usage of imperative verbs (Create, Delete) instead of using more open words such as (OK, No, Yes). Also, try to connect the Title wording to the CTA, to keep the action clear to the user, we recommend avoiding using more than two words.
+
+And regarding the difference of the types of variables of the component:
+
+- <b>Zero state</b>: Like we said in the uses and best practices, we recommend combining the title and the CTA (if usable and possible) to teach the user how to populate this area. E.g
+Title: You don’t have any AI recaps yet
+Description: Use it to analyze your calls and get more insights about it!
+CTA: [Activate]
+- <b>No results</b>: In this scenario, we need to contemplate the variables of the user to understand how to react, this could be a expected scenario (searching for something you know you won’t find) or the surprised scenario (‘I was expecting to find something here, but there’s nothing’)
+Title: No results found
+Description: Try adjusting the your search or filters to find what you are looking for
+CTA: [create] [clear filters]
+- <b>New feature</b>: We want to communicate this variable to be as easy to understand as possible. Usually when we communicate with this variable, we are offering a new service (paid or not) that the user can take advantage of, so work on selling it in a more marketable way.
+Title: Now you can analyze your metrics in live calls!
+Description: Don’t need to end a call to have the best insights.
+CTA: [Activate]
+- <b>Not available</b>: The user doesn’t have permission to see the information for X or Y. Try to use this space to make it easier to request permission to access. Be clear and concise to communicate that they don’t have access, and don’t spend too much time telling them why, focus on asking for permission.
+Title: You don’t have access to this information
+Description: Ask your administrators permission
+CTA: Request permission
+
+## Variants and examples
+There are some rules to follow when using empty state.
+
+- Consumer should <b><u>always</u></b> provide either a <b>bodyText</b> prop or content in <b>body</b> slot.
+- Console warning if both <b>bodyText</b> prop or <b>body</b> slot are not provided
+
+<br>
+On variable size:
+- Size only changes <b>headerText</b> and <b>bodyText</b> font size and width container.
+- Illustration component has priority over icon in <b>lg</b> and <b>md</b> size.
+- Illustration component will not be shown in <b>sm</b> size.
+- Icon component will be shown in <b>lg</b> and <b>md</b> size only if illustrationName prop is not provided.
+- Icon component will always be shown in <b>sm</b> size.
```

Comment:
Hmmm these are not outputting correctly for some reason.. maybe needs a line break after the header?

![Screenshot 2024-05-28 at 3 30 17 PM](https://github.com/dialpad/dialtone/assets/64808812/2331c3ea-db72-4892-9acb-71c3748e9dae)

Reply:
Oh wait.. I think this just needs to be deleted. It is a repeat of whats under the `## Variable size` section.

Still can put a line break after `## Variants and examples`